CMakeLists.txt

```diff
@@ -36,25 +36,21 @@ set (P4C_LIB_DEPS)
 find_package (PythonInterp REQUIRED)
 find_package (FLEX REQUIRED)
 find_package (BISON REQUIRED)
-find_package(Protobuf 3.0.0 REQUIRED)
+find_package (Protobuf 3.0.0 REQUIRED)
 find_package (Boost REQUIRED COMPONENTS iostreams)
+find_package (LibGc REQUIRED)
+find_package (LibGmp REQUIRED)
 include_directories(${Boost_INCLUDE_DIRS})
+include_directories(${LIBGMP_INCLUDE_DIR})
+include_directories(${LIBGC_INCLUDE_DIR})
 set (HAVE_LIBBOOST_IOSTREAMS 1)
-set (P4C_LIB_DEPS "${P4C_LIB_DEPS};${Boost_LIBRARIES}")
+set (HAVE_LIBGC 1)
+set (HAVE_LIBGMP 1)
+set (HAVE_LIBGMPXX ${LIBGMPXX_LIBRARY})
```

Comment:
is HAVE_LIBGMP or HAVE_LIBGMPXX used anywhere? they're not in cmake/config.h and I don't see them anywhere in the source code.